.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
   - 2.1.0
+env:
+  - JIRA_URL="jira.example.org"
```

Comment:
if you add a `global:`, only one build will be kicked off with all of the ENV vars set
eg.
```
env:
  global:
    - JIRA...
    - JIRA...
```
source - http://docs.travis-ci.com/user/environment-variables/